reana_server/reana_admin.py

```diff
@@ -589,11 +619,15 @@ def set_quota_limit(ctx, admin_access_token, emails, resource_name, limit):
 def set_default_quota_limit(ctx, admin_access_token):
     """Set default quota limits to users that do not have any."""
     users_without_quota_limits = User.query.filter(~User.resources.any()).all()
+    if not users_without_quota_limits:
+        click.secho("There are no users without quota limits.", fg="green")
+        sys.exit(1)
```

Comment:
We might not need to do exit with error code 1 as this is not really an error but some information. What about `sys.exit(0)` instead?